src/library/entities/Library.java

```diff
@@ -84,163 +84,163 @@ public static synchronized void SaVe() {
+		member.takeOutLoan(loan); //Changed TaKe_OuT_LoAn to takeOutLoan according to method in Member
 		book.BoRrOw();
-		LoAnS.put(loan.GeT_Id(), loan);
-		CuRrEnT_LoAnS.put(book.gEtId(), loan);
+		loans.put(loan.GeT_Id(), loan);
+		currentLoans.put(book.gEtId(), loan);
 		return loan;
 	}
 	
 	
-	public Loan GeT_LoAn_By_BoOkId(int bookId) {
-		if (CuRrEnT_LoAnS.containsKey(bookId)) 
-			return CuRrEnT_LoAnS.get(bookId);
-		
+	public Loan getLoanByBookId(int bookId) { //Changed GeT_LoAn_By_BoOkId to getLoanByBookId
+		if (currentLoans.containsKey(bookId)) {
+			return currentLoans.get(bookId);
+		} //Added curly brackets	
 		return null;
 	}
 
 	
-	public double CaLcUlAtE_OvEr_DuE_FiNe(Loan LoAn) {
-		if (LoAn.Is_OvEr_DuE()) {
-			long DaYs_OvEr_DuE = Calendar.gEtInStAnCe().GeT_DaYs_DiFfErEnCe(LoAn.GeT_DuE_DaTe());
-			double fInE = DaYs_OvEr_DuE * FiNe_PeR_DaY;
-			return fInE;
+	public double calculateOverdueFine(Loan loan) { //Changed CaLcUlAtE_OvEr_DuE_FiNe to calculateOverdueFine, changed LoAn to loan
+		if (loan.Is_OvEr_DuE()) {
+			long daysOverdue = Calendar.getInstance().getDaysDifference(loan.GeT_DuE_DaTe()); //Changed GeT_DaYs_DiFfErEnCe to getDaysDifference according to method in Calendar
+			double fine = daysOverdue * FINE_PER_DAY; //Changed fInE to fine, changed DaYs_OvEr_DuE to daysOverdue
+			return fine;
 		}
 		return 0.0;		
 	}
 
 
-	public void DiScHaRgE_LoAn(Loan cUrReNt_LoAn, boolean iS_dAmAgEd) {
-		Member mEmBeR = cUrReNt_LoAn.GeT_MeMbEr();
-		Book bOoK  = cUrReNt_LoAn.GeT_BoOk();
+	public void dischargeLoan(Loan currentLoan, boolean isDamaged) { //Changed DiScHaRgE_LoAn to dischargeLoan, cUrReNt_LoAn to currentLoan, iS_dAmAgEd to isDamaged
+		Member member = currentLoan.GeT_MeMbEr(); //Changed mEmBeR to member
+		Book book  = currentLoan.GeT_BoOk(); //Changed bOoK to book
 		
-		double oVeR_DuE_FiNe = CaLcUlAtE_OvEr_DuE_FiNe(cUrReNt_LoAn);
-		mEmBeR.AdD_FiNe(oVeR_DuE_FiNe);	
+		double overdueFine = calculateOverdueFine(currentLoan); //Changed oVeR_DuE_FiNe to overdueFine
+		member.addFine(overdueFine);	//Changed AdD_FiNe to addFine according to method in Member
 		
-		mEmBeR.dIsChArGeLoAn(cUrReNt_LoAn);
-		bOoK.ReTuRn(iS_dAmAgEd);
-		if (iS_dAmAgEd) {
-			mEmBeR.AdD_FiNe(damageFee);
-			DaMaGeD_BoOkS.put(bOoK.gEtId(), bOoK);
+		member.dischargeLoan(currentLoan); //Changed dIsChArGeLoAn to dischargeLoan according to method in Member
+		book.ReTuRn(isDamaged);
+		if (isDamaged) {
+			member.addFine(DAMAGE_FEE);
+			damagedBooks.put(book.gEtId(), book);
 		}
-		cUrReNt_LoAn.DiScHaRgE();
-		CuRrEnT_LoAnS.remove(bOoK.gEtId());
+		currentLoan.DiScHaRgE();
+		currentLoans.remove(book.gEtId());
 	}
 
-
-	public void cHeCk_CuRrEnT_LoAnS() {
-		for (Loan lOaN : CuRrEnT_LoAnS.values()) 
-			lOaN.cHeCk_OvEr_DuE();
-				
+	
+	public void checkCurrentLoans() { //Changed cHeCk_CuRrEnT_LoAnS to checkCurrentLoans
+		for (Loan loan : currentLoans.values()) //Changed lOaN to loan
+			loan.cHeCk_OvEr_DuE();			

```

Comment:
For Loop Needs Curly Brackets

Reply:
Added curly brackets in the latest commit